blocks/video/video.js

```diff
@@ -4,14 +4,36 @@
  * https://www.hlx.live/developer/block-collection/video
  */
 
+const getVideoElement = (source, autoplay) => {
+  const video = document.createElement('video');
+  video.setAttribute('controls', '');
+  if (autoplay) video.setAttribute('autoplay', '');
+
+  const sourceEl = document.createElement('source');
+  sourceEl.setAttribute('src', source);
+  sourceEl.setAttribute('type', `video/${source.split('.').pop()}`);
+  video.append(sourceEl);
+
+  return video;
+};
+
 export default async function decorate(block) {
   const a = block.querySelector('a');
   if (a) {
     const source = a.href;
-    block.innerHTML = `
-    <video controls>
-      <source src="${source}" type="video/${source.split('.').pop()}" >
-    </video>
-    `;
+    const pic = block.querySelector('picture');
+    block.innerHTML = '';
+    if (pic) {
+      const wrapper = document.createElement('div');
+      wrapper.className = 'video-placeholder';
+      wrapper.innerHTML = '<div class="video-placeholder-play"><button title="Play"></button></div>';
```

Comment:
recommending [`type="button"`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#type),

Reply:
Also added this to embed block for consistency.